src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
  * Automatically register kernel methods marked with the `Symfony\Component\Routing\Annotation\Route` attribute or annotation as controllers in `MicroKernelTrait`
  * Deprecate not setting the `http_method_override` config option. The default value will change to `false` in 7.0.
  * Add `framework.profiler.collect_serializer_data` config option, set it to `true` to enable the serializer data collector and profiler panel
+ * Set locale from request rather than default locale if `framework.enabled_locales` is empty and `framework.set_locale_from_accept_language` is true
```

Comment:
should be removed, that's a bugfix

Reply:
Okay, was not sure. That’s why I opened the first PR against 6.2